RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -29,6 +31,7 @@ public AssignedByValParameterMakeLocalCopyQuickFix(Declaration target, Qualified
             _dialogFactory = dialogFactory;
             _parserState = parserState;
             _forbiddenNames = parserState.DeclarationFinder.GetDeclarationsWithIdentifiersToAvoid(target).Select(n => n.IdentifierName);
+            _setRequirementEvaluator = new VariableRequiresSetAssignmentEvaluator(_parserState);
         }
 
         public override bool CanFixInModule => false;
```

Comment:
Is there a reason the quick-fix couldn't be applicable to a whole module (or project)?